api/capi/include/nnstreamer.h

```diff
@@ -569,6 +574,67 @@ int ml_pipeline_valve_release_handle (ml_pipeline_valve_h valve_handle);
  */
 int ml_pipeline_valve_set_open (ml_pipeline_valve_h valve_handle, bool open);
 
+/********************************************************
+ ** NNStreamer Element Property Control in Pipeline    **
+ ********************************************************/
+
+/**
+ * @brief Gets an element handle in NNStreamer pipelines to control its properties.
+ * @since_tizen 6.0
+ * @remarks If the function succeeds, @a elm_h handle must be released using ml_pipeline_release_element().
+ * @param[in] pipe The pipeline to be managed.
+ * @param[in] element_name The name of element to control.
+ * @param[out] elm_h The element handle.
+ * @return @c 0 on success. Otherwise a negative error value.
+ * @retval #ML_ERROR_NONE Successful
+ * @retval #ML_ERROR_NOT_SUPPORTED Not supported.
+ * @retval #ML_ERROR_INVALID_PARAMETER Given parameter is invalid.
+ * @retval #ML_ERROR_OUT_OF_MEMORY Failed to allocate required memory.
+ */
+int ml_pipeline_get_element (ml_pipeline_h pipe, const char *element_name, ml_pipeline_element_h *elm_h);
```

Comment:
How about naming `ml_pipeline_element_get_handle` or `ml_pipeline_elem_get_handle` which will be inline with already corresponding functions of src?
Further, would it better to have this function consume existing functions such as `ml_pipeline_*_get_handle` etc?

Reply:
> How about naming ml_pipeline_element_get_handle or ml_pipeline_elem_get_handle which will be inline with already corresponding functions of src?

You are right. The name needs to be updated. 


> Further, would it better to have this function consume existing functions such as ml_pipeline_*_get_handle etc?

You means `ml_pipeline_src_get_handle()` or `ml_pipeline_switch_get_handle()`. right?

In my opinion, the main purpose of these APIs is for general GstElements except AppSrc, AppSink, TensorSink, Selector, and Valve. Special GstElements like AppSrc already has its control function such as `ml_pipeline_src_input_data()`. Because of this reason, I do not use the existing `handle` functions. 

I will check others' opinions on whether Special GstElements should be supported or not.

Reply:
I meant a common `get_handle()` which can handle all types rather than having multiple ones.
yes, as you said, let's wait for other's opinions.